app/recaptcha/client/captcha.py

```diff
@@ -27,92 +29,71 @@ def get_display_html(public_key, use_ssl=False, error=None,
 
     error_param = ''
     if error:
-        error_param = '&error=%s' % error
+        error_param = 'error=%s' % error
     server = API_SERVER
     if use_ssl:
         server = API_SSL_SERVER
 
-    # _('...') used to return objects that are unpalatable to simplejson.
-    # For better compatibility, we keep this conversion code, but execute it
-    # only when values are non-unicode to prevent UnicodeEncodeError.
-    if any(not isinstance(v, unicode) for v in custom_translations.values()):
-      custom_translations = dict((k, unicode(str(v), 'utf-8'))
-                                 for (k, v) in custom_translations.items())
-
-    options = {
-        'theme': 'white',
-        'lang': lang,
-        'custom_translations': custom_translations
-    }
-
-    return '''
-<script>
-  var RecaptchaOptions = %(options)s;
-</script>
-<script src="%(server)s/challenge?k=%(public_key)s%(error_param)s"></script>
-
+    html = '''
+<script src='%(server)s.js?hl=%(lang)s'></script>
+<div class='g-recaptcha' data-sitekey='%(public_key)s'></div>
 <noscript>
-  <iframe src="%(server)s/noscript?k=%(public_key)s%(error_param)s"
-      height="300" width="500" frameborder="0"></iframe><br>
-  <textarea name="recaptcha_challenge_field" rows="3" cols="40"></textarea>
-  <input type="hidden" name="recaptcha_response_field" value="manual_challenge">
-</noscript>
+  <div style='width: 302px; height: 422px;'>
+    <div style='width: 302px; height: 422px; position: relative;'>
+      <div style='width: 302px; height: 422px; position: absolute;'>
+        <iframe src='%(server)s/fallback?k=%(public_key)s'
+                frameborder="0" scrolling="no"
+                style='width: 302px; height:422px; border-style: none;'>
+        </iframe>
+      </div>
+
+    </div>
+      <div style='width: 300px; height: 60px; border-style: none;
+                  bottom: 12px; left: 25px; margin: 0px; padding: 0px; right: 25px;
+                  background: #f9f9f9; border: 1px solid #c1c1c1; border-radius: 3px;'>
+        <textarea id="g-recaptcha-response" name="g-recaptcha-response"
+                  class="g-recaptcha-response"
+                  style="width: 250px; height: 40px; border: 1px solid #c1c1c1;
+                         margin: 10px 25px; padding: 0px; resize: none;">
+        </textarea>
+      </div>
+  </div>
+</noscript><br/><br/><br/>
 ''' % {
-    'options': simplejson.dumps(options),
     'server': server,
+    'lang': lang,
     'public_key': public_key,
-    'error_param': error_param,
 }
+    if error:
+        return '''
+        <div>%(error_param)s</div>
+        ''' % {
+            'error_param': error_param
```

Comment:
Do you think you need "error=" in the error message? If not, you can just output *error* here instead of error_param.

Also, please apply HTML escaping just in case. You can use django.utils.html.escape().